src/js/tabs/osd.js

```diff
@@ -449,6 +449,12 @@ OSD.formatPidsPreview = function(axis) {
     }
 };
 
+OSD.generateTxUplinkPowerPreview = function() {
+    const crsfIndex = FC.getSerialRxTypes().findIndex(name => name === 'CRSF');
+    const isXF = crsfIndex === FC.RX_CONFIG.serialrx_provider;
+    return FONT.symbol(SYM.RSSI) + (isXF ? '250MW' : '---');
```

Comment:
This doesn't match the behavior of the firmware change. First we should not unnecessarily constrain this to CRSF in the code. That would mean we'd need yet another Configurator change if this capability is added to another radio protocol in the firmware. Also the firmware does not display "---". It would display "0MW" for a protocol that doesn't currently support the uplink power.

Reply:
Thanks. Done.